recipes/r-nleqslv/meta.yaml

```diff
@@ -13,46 +10,28 @@ source:
   url:
     - https://cran.r-project.org/src/contrib/{{ name }}_{{ version }}.tar.gz
     - https://cran.r-project.org/src/contrib/Archive/{{ name }}/{{ name }}_{{ version }}.tar.gz
-
-
-  # You can add a hash for the file here, like md5 or sha1
-  # md5: 49448ba4863157652311cc5ea4fea3ea
-  # sha1: 3bcfbee008276084cbb37a2b453963c61176a322
-  # patches:
-   # List any patch files here
-   # - fix.patch
+  md5: a80cb89e109cb2547dd1ed1e397385de
 
 build:
-  # If this is a new build for the same version, increment the build number.
   number: 0
-
-  # This is required to make R link correctly on Linux.
   rpaths:
     - lib/R/lib/
     - lib/
 
-
 requirements:
   build:
     - r-base
-    - posix                # [win]
-    - {{native}}toolchain  # [win]
-    - gcc                  # [not win]
+    - posix
+    - {{native}}toolchain
```

Comment:
can remove the posix and native toolchain lines, since they were selected just for windows